rllib/env/vector_env.py

```diff
@@ -232,11 +256,34 @@ def reset_at(self, index: Optional[int] = None) -> EnvObsType:
             index = 0
         return self.envs[index].reset()
 
+    @override(VectorEnv)
+    def restart_at(self, index: Optional[int] = None) -> None:
+        if index is None:
+            index = 0
+        old_env = self.envs[index]
+        # Re-create the sub-env at the new index.
+        self.envs[index] = self.make_env(index)
+        # Try closing down the old (possibly faulty) sub-env, but ignore errors.
+        try:
+            old_env.close()
+        except Exception:
+            pass
```

Comment:
although we ignore the exception, I think it's better to still print a log msg.

Reply:
done